docs/reference/migration/migrate_5_0/plugins.asciidoc

```diff
@@ -112,3 +112,8 @@ Previously, Java system properties could be passed to the plugin
 command by passing `-D` style arguments directly to the plugin script.
 This is no longer permitted and such system properties must be passed
 via ES_JAVA_OPTS.
+
+==== Custom plugins path
+
+The ability to specify a custom plugins path via `path.plugins` has
+been removed.
```

Comment:
Maybe tell users that they can still just use a symlink if they really really want to keep things in a different place?